frontend/src/app/shared/components/colors/colors.service.ts

```diff
@@ -9,7 +9,10 @@ export const colorModes = {
 @Injectable({ providedIn: 'root' })
 export class ColorsService {
   public toHsl(value:string, colorMode?:string):string {
-    return `hsl(${this.valueHash(value)}, 50%, ${colorMode === colorModes.lightHighContrast ? '30%' : '50%'})`;
+    if (colorMode) {
+      return `hsl(${this.valueHash(value)}, 50%, ${colorMode === colorModes.lightHighContrast ? '30%' : '50%'})`;
+    }
+    return `hsl(${this.valueHash(value)}, 50%, 30%)`;
```

Comment:
This is somehow strange.. For one, you are doubling cod. Second, the default fallback is the same as the high contrast mode, however when I pass a mode (even the default "light" one) I get a different result than without passing it... Since you also removed the `colorMode` from being passed to the method I guess we can get rid of it completely.